Max simultaneous backups defaults to 4 (#2549)
RedisLabs/redislabs-docs: content/rc/databases/back-up-data.md

- Free plans cannot perform backups through the Redis Cloud console.

{{<note>}}

> The number of database backups that can run simultaneously on a cluster node is limited to 4 by default.

Reviewer: ```suggestion
The number of simultaneous database backups running on a cluster node is limited to 4 by default.
```

> The number of database backups that can run simultaneously on a cluster node is limited to 4 by default.

Author: ```suggestion
The number of database backups that can run simultaneously on a cluster is limited to 4 by default.
```
